tests/spec/core/xref-spec.js

```diff
@@ -156,10 +157,10 @@ describe("Core — xref", () => {
     // https://github.com/w3c/respec/pull/1750
     const body = `
       <section id="test">
-        <p data-cite="css-values"><a id="one">ident</a></p>
+        <p data-cite="css-values"><a id="one" data-xref-type="css-value" data-xref-for="attr()">ident</a></p>
         <p data-cite="css-syntax"><a id="two">ident</a></p>
         <p data-cite="css-syntax">
-          <a id="three" data-cite="css-values">ident</a> (overrides parent)
+          <a id="three" data-cite="css-values" data-xref-for="CSS">ident</a> (overrides parent)
```

Comment:
Why are these suddenly needed? 👀

Reply:
They got updated in the new spec: 
https://respec.org/xref/?term=ident 

In level 3, it was just [=ident=], but in 4, it's [=CSS/ident=].